packages/sdk/src/types/Redemptions.ts

```diff
@@ -194,16 +194,20 @@ export type RedemptionsRedeemStackableRedemptionResult = {
 	failure_message?: string
 }
 
-export type RedemptionsRedeemStackableOrderResponse = OrdersCreateResponse & {
+export type RedemptionsRedeemStackableOrderResponse = Omit<OrdersCreateResponse, 'source_id'> & {
+	source_id: string | null
+	customer_id: string | null
+	referrer_id: string | null
 	redemptions?: Record<
 		string,
 		{
 			date: string
 			rollback_id?: string
 			rollback_date?: string
-			related_object_type: 'redemption'
+			related_object_type: 'redemption' | 'voucher' | 'promotion_tier'
```

Comment:
a promotion_stack nie?

Reply:
dokumentacja nic nie mówi o takim unionie niestety, więc nie wiem

Reply:
Chodziło o to byś sprawdził, robiąc redeema z promotion stackiem:
Ja teraz zrobiłem i jest:
![Screenshot 2024-11-28 at 14 14 01](https://github.com/user-attachments/assets/79e21864-6dc9-4a2a-8cba-803372fc4047)
więc proszę dodaj `promotion_stack` jako możliwość